### DISCUSSION TITLE
Crawling Tree List Categories Filters

### DISCUSSION THREAD

**Discussion by Moskitoz** [projectdiscovery/katana#244] [Ideas]
Dear developers,,
Thanks a lot for this Great Tool

It's good to add some feathers to make this tool scan Tree List Categories Filters

Example
https://i.ibb.co/RHyx7p6/filters.png

This menu Filter add some filter parameters to show result on the page
when you select a filter from the list, the parameter will be added to the URL

some of this Filter List have requests with GET requests 
and some comes with POST requests

from my opinion it's good to add this feather to make the tool discover more parameters on the Targets

Thank you in Advance to the Developers
Regards,,

**Comment by MetzinAround** [comment]
Any idea on whether this is something we'd be able to do, @ehsandeep ?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
